akka-docs-dev/rst/scala/code/docs/utils/TestUtils.scala

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package docs.utils
+
+import java.net.InetSocketAddress
+import java.nio.channels.ServerSocketChannel
+
+object TestUtils {
+  def temporaryServerAddress(interface: String = "127.0.0.1"): InetSocketAddress = {
```

Comment:
No wait. We depend on published versions of akka artifacts in `release-2.3-dev` branch. Probably the way forward to remove code duplication is https://github.com/akka/akka/issues/17080